cmake/modules/LLDBConfig.cmake

```diff
@@ -52,6 +52,10 @@ option(LLDB_BUILD_FRAMEWORK "Build LLDB.framework (Darwin only)" OFF)
 option(LLDB_NO_INSTALL_DEFAULT_RPATH "Disable default RPATH settings in binaries" OFF)
 option(LLDB_USE_SYSTEM_DEBUGSERVER "Use the system's debugserver for testing (Darwin only)." OFF)
 
+# BEGIN SWIFT MOD
+option(LLDB_ENABLE_SWIFT_SUPPORT "Enable swift support" ON)
```

Comment:
It might be a good idea to do some error checking here. Specifically, if LLDB_ENABLE_SWIFT_SUPPORT is ON but we can't find swift, then throw an error.